### ISSUE TITLE
cvbio runs into error with recent openjdk version

### ISSUE THREAD

**Issue by ManavalanG** [clintval/cvbio#89]
When I installed cvbio v3.0.0 via bioconda, without specifying openjdk version, it installed successfuly with openjdk v17.0.3. However when executed, cvbio ran into this error: 

```
Unrecognized VM option 'AggressiveOpts'
Error: Could not create the Java Virtual Machine.
Error: A fatal exception has occurred. Program will exit.
```

When installed with openjdk v11.0.15, cvbio ran successfully but it produced this warning message

```
OpenJDK 64-Bit Server VM warning: Option AggressiveOpts was deprecated in version 11.0 and will likely be removed in a future release.
```

**Comment by clintval** [comment]
Thank you for reporting! Seems like it could be an easy fix.
